packages/legend-query-builder/src/components/data-access/DataAccessOverview.tsx

```diff
@@ -41,10 +41,13 @@ import {
   DatasetEntitlementAccessRequestedReport,
   DatasetEntitlementUnsupportedReport,
 } from '@finos/legend-graph';
+import { Chart as ChartJS, DoughnutController, ArcElement } from 'chart.js';
 import { Doughnut } from 'react-chartjs-2';
 import { getNullableFirstEntry } from '@finos/legend-shared';
 import type { QueryBuilder_LegendApplicationPlugin_Extension } from '../../stores/QueryBuilder_LegendApplicationPlugin_Extension.js';
 
+ChartJS.register(DoughnutController, ArcElement);
```

Comment:
Would you mind explaining to me what it is for ? Thanks!

Reply:
It turns out you need to register the components used in the chart or they might get pruned during treeshaking optimizations: https://stackoverflow.com/questions/70098392/react-chartjs-2-with-chartjs-3-error-arc-is-not-a-registered-element

So this change just registers the 2 components (`DoughnutController` and `ArcElement`) so they are always available. Otherwise, when we render this component in the VS Code extension, it throws this error: `Uncaught Error: "arc" is not a registered element.`